spec/colorizer.rb

```diff
@@ -20,7 +20,7 @@
       {rubymotion: "rocks!"},
       Time.now,
       Object.new,
-      UIView.new,
+      NSSet.new,
```

Comment:
At first I was wondering why you took out UIView in the test and then realized it doesn't exist on Mac :)